app/controllers/dailyhabits_controller.rb

```diff
@@ -5,14 +5,19 @@ def update
     habit = current_user.dailyhabits.find(params[:id])
     if habit.completed_at.nil?
       score = habit.streak_count
-      habit_count = Dailyhabit.where(user_id: current_user.id).where('date >= ?', DateTime.now.beginning_of_day..DateTime.now.end_of_day).count
-      daily_point_value = ( 100.0 / habit_count.to_f )
+      habit_count = Dailyhabit.where(
+        user_id: current_user.id).where(
+        'date >= ?', DateTime.now.beginning_of_day..DateTime.now.end_of_day
```

Comment:
Indent the first parameter one step more than the previous line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.